CHANGELOG.md

```diff
@@ -537,7 +539,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Minor fixes and improvements on features and tests
 
-[unreleased]: https://github.com/openfun/marsha/compare/v3.9.0...master
+[unreleased]: https://github.com/openfun/marsha/compare/v3.9.1...master
+[3.9.0]: https://github.com/openfun/marsha/compare/v3.9.0...v3.9.1
```

Comment:
Ah! Got you!

Reply:
```suggestion
[3.9.1]: https://github.com/openfun/marsha/compare/v3.9.0...v3.9.1
```

Reply:
![](https://media.giphy.com/media/fniOPx1Sf5QNjh0yqQ/giphy.gif)